File: catalog/admin/item/ordering.php
<?php
defined('AUTH') or die('Restricted access');

include_once $_SERVER['DOCUMENT_ROOT'].'/components/catalog/admin/classes/AdminCatalogItem.php';
$ITEM = new AdminCatalogItem();

$arr['id'] = intval($SITE->d[5]);  // section id
$arr['type'] = $SITE->d[4];  // up / down

$section_id = $ITEM->getItem($arr['id'])['section_id'];

$ITEM->ordering($arr);

header('location: /admin/com/catalog/section/'.$section_id);

File: dan/system/catalog/item/ordering.py
import sys
sys.path.append('system/catalog/classes')
from system.catalog.classes.Item import Item


def ordering(SITE):
    print('FUNCTION -> system-> calalog -> item -> ordering')
    type = SITE.p[2]
    id = SITE.p[3]

    ITEM = Item(SITE)
    section_id = ITEM.ordering(type, id)

    if 'cancel' in SITE.post:
        return {'redirect': '/system/catalog/section/' + str(section_id)}
    return {'redirect': '/system/catalog/section/' + str(section_id)}
